packages/cascara/src/structures/Admin/AdminDrawer.js

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 import pt from 'prop-types';
 import styles from './Admin.module.scss';
+import { Role } from 'reakit/Role';
+import classnames from 'classnames/bind';
 
 const propTypes = {
+  as: pt.string,
   children: pt.oneOfType([pt.node, pt.array]),
+  className: pt.string,
 };
 
-const AdminDrawer = ({ children, ...rest }) => {
+const cx = classnames.bind(styles);
+
+const AdminDrawer = ({ as = 'div', className, children, ...rest }) => {
   return (
-    <div {...rest} className={styles.Drawer}>
+    <Role {...rest} as={as} className={cx(className, 'Drawer')}>
```

Comment:
We will talk about this at the end of standup... but I think we need to make sure we are consistent on these items in terms of what order we are doing the rest spread, and what order we are defining the className versus the initial class. Both of these can potentially impact functionality so we should be consistent with what order we define them. I have conflicted feelings for each so we should discuss and come to a consensus.

Reply:
I'll create a note to discuss this in our next standup

Reply:
Let's define the internal class first (`'Drawer'`), then `rest.className` and remove className from our propTypes.